src/scrabbleScore.js

```diff
@@ -24,14 +24,20 @@ let SCRABBLE_TILE_SCORES = {
  */
 
 function scrabbleScore(word) {
-  // This is your job. :)
+  let score = 0;
+  let wordArray = word.toLowerCase().split('');
+  for(let letter of wordArray){
```

Comment:
No need to convert `word` into an array first, you can iterate over the letters in a string directly, e.g.,

```javascript
for (let letter of word.toLowerCase()) {
  // ...
}
```